Makefile

```diff
@@ -171,37 +181,34 @@ run-kubernetes-master: stop-kubernetes-master
 
 	# Wait until the apiserver is accepting requests.
 	while ! docker exec st-apiserver kubectl get nodes; do echo "Waiting for apiserver to come up..."; sleep 2; done
+	sleep 5
```

Comment:
Why do we need another sleep if we already have the `while` loop to check that the api server is running?

Reply:
Yeah, I was meaning to remove this and see. 

It seemed like there was some extra delay in setting up the v1 CRDs compared to the v1beta1 CRDs that was causing some flakiness before. I'll retry without, but seems like some APIs become available faster than others.